### ISSUE TITLE
i want to use this SDK for watch App is it anyway i can use it for apple watch

### ISSUE THREAD

**Issue by ankushdhawan** [FuturraGroup/OpenAI#12]
I am getting this error (watchOS 9.0) is not compatible with OpenAIKit (1.9.3), which does not support watchOS.Please help me for this.do not close it without any expalination

**Comment by kyrylo-mukha** [comment]
Hello there. We've tested out and all works fine. Our SDK supports minimum version WatchOS 6.0.
We've tested out in minimum availeble WatchOS version on Xcode 15.3 - WatchOS 7.0 and all works fine.
Try to install library with SPM, check you are imported our library and check your code is correct. Sometimes Xcode highlight wrong errors not related to code in error.
Also, if you creating not standalone app for WatchOS try out [Watch Connectivity](https://developer.apple.com/documentation/watchconnectivity)
